TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -311,6 +327,22 @@ class RouteOptionsViewController: UIViewController {
             }
         }
     }
+    
+    @objc private func refreshRoutesAndTime() {
+        let now = Date()
+        lastRouteRefreshDate = now
```

Comment:
would move this down into the `searchForRoutes` to make sure it's updated *every* time we call `searchForRoutes`